quacc/settings.py

```diff
@@ -128,6 +128,38 @@ class QuaccSettings(BaseSettings):
         description="After this many seconds, Custodian will stop running and ensure that VASP writes a STOPCAR",
     )
 
+    # ---------------------------
+    # Q-Chem Settings
+    # ---------------------------
+
+    # Q-Chem Settings: Main
+    QChem_CMD: str = Field(
+        "qchem", description="Command to run the standard version of Q-Chem."
+    )
+
+    QChem_MAX_CORES: int = Field(
+        32, description="Maximum number of cores to use for the Q-Chem calculation."
```

Comment:
Could you explain what this does when you have a moment? It says "maximum," so is Custodian able to adjust the number of cores on-the-fly for some sort of error correction?

Reply:
That's a good question. When we named "qchem_max_cores" loooooong ago thinking that we may encounter cases where the number of cores would be dynamic. But in practice we never do, so this name would be clearer (at least in this code - I'm not changing it in Custodian) as QCHEM_CORES.

Reply:
Ah, I see. We can revisit this later, but my preference would be to mimic the ASE calculator style of adding the number of cores and such to the input arguments of the calculator (e.g. see [here](https://databases.fysik.dtu.dk/ase/ase/calculators/qchem.html)) for consistency rather than having this be some environment variable. This makes it easier to swap settings on-the-fly or for multiple jobs in a workflow.